src/doc/src/reference/config.md

```diff
@@ -67,6 +67,10 @@ incremental = true            # whether or not to enable incremental compilation
 dep-info-basedir = "…"        # path for the base directory for targets in depfiles
 pipelining = true             # rustc pipelining
 
+[cargo-doc]
```

Comment:
Looks like some of the documentation needs to be updated with the new table name.